puppeteer-test/src/tests/workspace/workspace-create.spec.ts

```diff
@@ -0,0 +1,44 @@
+import GoogleLoginPage from '../../pages/google-login';
+import Home from '../../pages/home';
+import PuppeteerLaunch from '../../services/puppeteer-launch';
+
+jest.setTimeout(60 * 1000);
+
+const configs = require('../../config/config');
+
+describe.skip('Workspace creation tests:', () => {
+
+  let browser;
+  let incognitoContext;
+  let page;
+
+  beforeAll(async () =>  {
+    browser = await PuppeteerLaunch();
+  });
+
+  beforeEach(async () => {
+    incognitoContext = await browser.createIncognitoBrowserContext();
+    page = await incognitoContext.newPage();
+    await page.setUserAgent(configs.puppeteerUserAgent);
+    await page.setDefaultNavigationTimeout(60000);
+  });
+
+  afterEach(async () => {
+    await incognitoContext.close();
+  });
+
+  afterAll(async () => {
+    await browser.close();
+  });
+
+  test('Create a new workspace from the Home page', async () => {
+    const loginPage = new GoogleLoginPage(page);
```

Comment:
I think it might be meaningful to create a helper function that handles Google login automatically, and simply allows a test case to say "Get me to page X, using whatever auth flow is necessary". This may help reduce boilerplate and allow us flexibility to swap-out the user/email password entry with an alternate "backdoor" accessToken auth mechanism in the future.